Makefile

```diff
@@ -661,12 +661,23 @@ eve-%: pkg/%/Dockerfile build-tools $(RESCAN_DEPS)
 	$(QUIET)$(LINUXKIT) $(DASH_V) pkg $(LINUXKIT_PKG_TARGET) $(LINUXKIT_OPTS) pkg/$*
 	$(QUIET): "$@: Succeeded (intermediate for pkg/%)"
 
-images/rootfs-%.yml.in: images/rootfs.yml.in FORCE
+images/rootfs-%.yml.in: images/rootfs.yml.in images/rootfs-%.yml.in.new FORCE
 	@if [ -e $@.patch ]; then patch -p0 -o $@.sed < $@.patch ;else cp $< $@.sed ;fi
 	$(QUIET)sed -e 's#EVE_VERSION#$(ROOTFS_VERSION)-$*-$(ZARCH)#' < $@.sed > $@ || rm $@ $@.sed
 	@rm $@.sed
+	@if ! diff -u $@ $(word 2,$^); then >&2 echo "new template doesn't match to the old"; false; fi
 	$(QUIET): $@: Succeeded
 
+# TODO: the new way of generating templates allows to combine
+# different flavours together (if compatible). This will be used later
+# by development builds, once the necessary changes are merged in
+# linuxkit. For now we keep both ways, and compare the output, to make
```

Comment:
@zededa-yuri here is the reference to changing linuxkit.

Reply:
right, but that's the next steps. For this patch none of that is required yet.

Reply:
What are the plans for changes?

Reply:
Development builds. What I currently have in my drafts would require to change `PILLAR_DEV_TAG` in the `rootfs.yml`, which would effect in using pillar with debug symbols, disabled optimisations, and delve binary included.

Unfortunately it's not possible to build a separate Pillar image without [my patch in linuxkit](https://github.com/linuxkit/linuxkit/pull/3774). So it needs to wait while you, @deitch , are finishing your efforts on chain builds so we can migrate to newer linuxkit.

An alternative could be building ourselves a custom linuxkit for the meanwhile. Which is already what I am doing in my draft.

Now I am thinking I will update this PR to actually introduce the development build, even though enabling it would fail the builds (because of the missing Pillar's image). This way the intention of the pull request would be a bit more clear.